doc/api/stream.md

```diff
@@ -1263,6 +1265,27 @@ implementors should not override this method, but instead implement
 [`readable._destroy`][readable-_destroy].
 The default implementation of `_destroy` for `Transform` also emit `'close'`.
 
+#### Class Method: stream.pipeline(...streams[, callback])
+<!-- YAML
+added: REPLACEME
+-->
```

Comment:
these are not class methods. See https://nodejs.org/api/http.html on hot to structure these.
Also the number of `#` is wrong.